### ISSUE TITLE
latest install revision

### ISSUE THREAD

**Issue by thisledome5** [socfortress/CoPilot#148]
Hello @taylorwalton 
In your latest revision guide here, It has a single line to bring down the docker-compose.yml

which when run, fails,
You say to modify the compose file to add server name and services
but that immediately errors out and says mapping values are not allowed here

I don't get it actually

You went from a complete front end back end install to the below commans which. don't appear to work.
Or am I missing something. which could be true also.

#  Clone the CoPilot repository
wget https://raw.githubusercontent.com/socfortress/CoPilot/v0.0.2/docker-compose.yml

# Edit the docker-compose.yml file to set the server name and/or the services you want to use

# Create the path for storing your data
mkdir data

# Run Copilot
docker compose up -d

**Comment by thisledome5** [comment]
And so with more time to mull over the issues. I have rebuilt a fresh ubuntu 22:04 with all updates.
cloned the repository
cloned the new docker file and presto. I am back to where I started on the first build of copilot
error cant bind to IP
INFO:     Application startup complete.
ERROR:    [Errno 99] error while attempting to bind on address ('208.87.135.101', 5000): cannot assign requested address
INFO:     Waiting for application shutdown.
2024-02-14 11:56:39.087 | INFO     | __main__:shutdown_scheduler:146 - Shutting down scheduler
2024-02-14 11:56:39.098 | INFO     | app.db.db_setup:ensure_scheduler_user_removed:118 - Ensuring scheduler user exists
2024-02-14 11:56:39.120 | INFO     | app.auth.services.universal:remove_scheduler_user:185 - Scheduler user removed.
INFO:     Application shutdown complete.

**Comment by taylorwalton** [comment]
@thisledome5 that docker-compose was outdated....can you try the latest now? also just set the `server_ip` to `0.0.0.0`

**Comment by thisledome5** [comment]
Hi @taylorwalton  Thanks for the correctinfo.
Ok so my issue is there is no complete guide to install CoPilot from scratch.
The new updated docker-compose update presumes that you have a complete install done
If another noob comes along, there is no doc guide to follow.  I found the video and had to copy the onscreen data down to install. then I had to git the new compose file. that was my problem. I had the wrong one. so now it is running with no errors.

Thanks and hopefully there will be  an always live install from scratch doc available soon.
keep up the good work.
I am creating another issue as I can not get it to connect to Wazuh.
